### DISCUSSION TITLE
Help with preventing whatever it is that is making some shows purple

### DISCUSSION THREAD

**Discussion by Silicon-Falcon** [RandomNinjaAtk/docker-sonarr-extended#41] [Q&A]
I know this has something to do with my nVidia Shield and HDR something or whatever.
I just want to stop downloading whatever type it is.
My install is the default docker install for Unraid.

**Comment by RandomNinjaAtk** [comment]
Sounds like your downloading HDR content and having difficulty playing it back... If that's the case, you'll need to change your configuration to avoid downloading it...

**Reply by Silicon-Falcon** [reply]
What is it that I need to change? Is it in Profiles, Quality, or in the custom formats?

**Reply by RandomNinjaAtk** [reply]
Depends on your config. You should look at these guides for config help:

https://trash-guides.info/

Also might be good to ask for help in official sonarr community/support channels.

**Reply by Silicon-Falcon** [reply]
alright, thanks anyway.
